app/src/main/java/ca/marshallasch/veil/FragmentLogin.java

```diff
@@ -46,12 +54,26 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         MaterialButton login = view.findViewById(R.id.enter_btn);
         MaterialButton cancel = view.findViewById(R.id.back_btn);
 
+
+        emailAddressInput = view.findViewById(R.id.username);
+        passwordInput = view.findViewById(R.id.password);
+
         login.setOnClickListener(view1 -> {
             Log.i("Fragment Login", "enter button pressed");
-            //TODO: check username and password
+
             hideKeyboard(view1);
-            ((MainActivity) getActivity()).navigateTo(new FragmentDashBoard(), true);
 
+            Database db = Database.getInstance(getActivity());
```

Comment:
Add in some comments about what the db calls are for.

Reply:
Fixed in commit 8f57e41